### ISSUE TITLE
Conversion from Gauss to Tesla seems to be strange

### ISSUE THREAD

**Issue by pasbec** [hgrecco/pint#1209]
Hi,

According to [Wikipedia](https://en.wikipedia.org/wiki/Gauss_(unit)) 10000 G (gauss) should be equal to 1 T (tesla) and share the dimensionality ([mass] / [current] / [time] ** 2).

However the conversion fails with pint:
```python
from pint import UnitRegistry

ureg = UnitRegistry()
Quantity = ureg.Quantity

Quantity(1, "G").to("T")
```
```txt
DimensionalityError: Cannot convert from 'gauss' ([mass] ** 0.5 / [length] ** 0.5 / [time]) to 'tesla' ([mass] / [current] / [time] ** 2)
```

It seems like Gauss has been given the dimensionality ([mass] ** 0.5 / [length] ** 0.5 / [time]) in pint!?

Is this "another" unit?

**Comment by jules-ch** [comment]
You can see how to use use Gaussian units here : #1205. 

@hgrecco , as you said we need to add something in the docs since it is confusing for users trying to use those units.

**Comment by jules-ch** [comment]
TLDR: 
```python
ureg​.​enable_contexts​(​"Gaussian"​)
```

**Comment by pasbec** [comment]
Thanks for the quick reply - guess I should have used the search before. :-)
So I'm the next who somewhat needs to understand CGS...

Do understand it right that if I enable Gaussian context that this switches pint to CGS?
Because that is exactly what I DON't want.

I thought the conversion 10000G = 1T should be true in the metric system?
So I probably have to redefine the unit.

Is this possible without fiddling with the pint files?

**Comment by pasbec** [comment]
This seems to work, too:
```python
import pint
from pint import UnitRegistry

ureg = UnitRegistry()
Quantity = ureg.Quantity

#ureg.enable_contexts("Gaussian")
ureg.define("Gauss = 1e-4 tesla = G")

Quantity(1, "G").to("T")
```
```txt
0.0001 tesla
```

**Comment by jules-ch** [comment]
No it won't switch to cgs it will allow you to use Gaussian units with Tesla and other units. It is the purpose of contexts

**Comment by pasbec** [comment]
> No it won't switch to cgs it will allow you to use Gaussian units with Tesla and other units. It is the purpose of contexts

OK, Thank you!

**Comment by jules-ch** [comment]
As I said in #1205 the BIPM is allowing Gauss units to work with the SI system. https://www.bipm.org/en/publications/si-brochure/table9 they made a note to be careful when comparing with those units. Overall stick with SI where you can.

**Comment by tovrstra** [comment]
@jules-ch When one just wants to have `Gauss` and `G` as a shorthand for `1e-4*Tesla` without activating all the other Gaussian units (because that seems like opening a can of worms), isn't then `ureg.define("Gauss = 1e-4 tesla = G")` the better option?
